src/modules/Api.js

```diff
@@ -0,0 +1,39 @@
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/0pNrfjBwCKXqbjF87mV/scores/';
+const addData = function (name, score) {
+  const DataToSend = {
+    user: name,
+    score,
+  };
+  fetch(url, {
+    method: 'post',
+    body: JSON.stringify(DataToSend),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  })
+    .then((res) => res.json())
+    .then((data) => data);
+};
+
+const catchData = async function () {
+  const data = await fetch(url);
+  const parsedData = await data.json();
+  return parsedData;
+};
+
+async function displayAllScore() {
```

Comment:
[REQUIRED] It is a requirement in this milestone for you to use arrow functions instead of the function keyword. Kindly fix this.